src/test/kotlin/SerializationTest.kt

```diff
@@ -0,0 +1,13 @@
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class TxIdsTest {
+    @Test
+    fun testDeserialize() {
+        val json = """{"txids":["txid1","txid2","txid3"]}"""
```

Comment:
What mempool space returns is something like:

```suggestion
        val json = """["txid1","txid2","txid3"]"""
```